content/post/20200426-kubernetes-the-universal-control-plane/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Kubernetes: The Universal Control Plane"
+date: 2020-04-26T23:02:29-07:00
+tags:
+  - Kubernetes
+  - cloud
+---
+
+> The innovation is in the API
+
+*This is the first in a 2 part series on Kubernetes, exploring what changes with the project in the picture. Part one is an answer: What are the key ideas that influence the design of Kubernetes? What about Kubernetes sets it apart from other platforms? Part two is about the second order effects: How does the industry respond to a universal control plane? What will our tooling, practices, and platforms look like in the future?*
```

Comment:
I'd remove "exploring what changes with the project in the picture"
I don't think it adds enough and just makes the intro a bit confusing